ReactWindows/ReactNative/UIManager/BaseViewManager.cs

```diff
@@ -210,9 +212,15 @@ public override void SetDimensions(TFrameworkElement view, Dimensions dimensions
             if (overflowHidden)
             {
                 SetOverflowHidden(view, dimensions);
+                view.SizeChanged -= OnSizeChanged;
             }
 
             base.SetDimensions(view, dimensions);
+
+            if (overflowHidden)
+            {
+                view.SizeChanged += OnSizeChanged;
+            }
```

Comment:
I'm not sure I like the approach of removing and readding event handlers. It is something a developer will forget to do if they add their own OnSizeChanged event or if they override this method. Is there a way to move this logic to the OnSizeChanged handler or whatever triggers the SizeChanged event?